graylog2-server/src/main/java/org/graylog2/audit/AuditEventTypes.java

```diff
@@ -319,6 +323,9 @@ public class AuditEventTypes implements PluginAuditEventTypes {
             .add(TELEMETRY_USER_SETTINGS_UPDATE)
             .add(CONTENT_STREAM_USER_SETTINGS_UPDATE)
             .add(CERTIFICATE_RENEWAL_MANUALLY_INITIATED)
+            .add(ENTITY_GROUP_CREATE)
+            .add(ENTITY_GROUP_UPDATE)
+            .add(ENTITY_GROUP_DELETE)
```

Comment:
It looks like formatters for these audit entries are still needed, since the general message is still being logged.
<img width="905" alt="image" src="https://github.com/Graylog2/graylog2-server/assets/3423655/fad07a19-e319-4253-adb6-eca192e83c1b">